### PR TITLE
fixed typo on documentation / typographical error

### PR DESCRIPTION
**Pull request by droxer**

### CODE REVIEW

**Comment by CaselIT** [comment]
Not a native English speaker, but to me the current version look correct.
I'll leave this one to you @jvanasco @zzzeek

**Comment by suesunss** [comment]
Hi guys, 

Recently my new project has adopted SQLAlchemy as the toolkit for database operations. 

I read a lot of official documentation and I found that, frankly speaking, the documentation is really poorly written, the paragraphs are poorly organized, and it feel like the language is not coming from a native English speaker.

From my experience, good examples are, e.g. elasticsearch's documentation.

Really wish the documentation could be better !

**Comment by CaselIT** [comment]
Hi @suesunss 
Not sure why this comment it in this pr, since it seems completely unrelated.

Your comment is a bit hard to act upon since it has no examples and it's not really an option rewiring the documentation from scratch... I can't say that this is a common complaint thought.

If you could open an issue with examples or a pr with how to improve them it would help.

**Comment by zzzeek** [comment]
> Hi guys,
> 
> Recently my new project has adopted SQLAlchemy as the toolkit for database operations.
> 
> I read a lot of official documentation and I found that, frankly speaking, the documentation is really poorly written, the paragraphs are poorly organized, and it feel like the language is not coming from a native English speaker. Most of the time, I feel like I'm missing the point.

well just for background 95% of the docs were written (and rewritten, and rewritten again, and again, for years, to make them better and better) by me and I don't even speak any other languages other than English, as is so common for we Americans.     My writing is more formalized than my spoken language because I've observed that if I speak more "off the cuff" then I use non-consistent terminology that makes things worse, however I'd be interested to know if you find it hard to follow how I speak in some of my talks, https://www.sqlalchemy.org/library.html#tutorials .  

seeing that the docs have been consumed by thousands of users for over a decade, and while we get a lot of complaints about various areas which we always act upon, as well as a huge amount of compliments about how our docs are amazing (this doesnt negate people not liking our docs, there's both camps out there for sure), this particular comment kind of takes the cake.    So, feel free to support your assertion with some specific places that we can go and fix up the grammar, though I would think the vast majority of the grammar is close to correct as I am pretty decent with grammar compared to the average Facebook poster.
